bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/LanguageNavigation.java

```diff
@@ -51,16 +47,7 @@
      * @since com.adobe.cq.wcm.core.components.models 12.2.0
      */
     default List<NavigationItem> getItems() {
-        throw new UnsupportedOperationException();
+        return null;
```

Comment:
Shouldn't we return an empty list as a general pattern (as done above)?

Reply:
I opted to return `null` for all methods (expect primitives and those annotated with `@NotNull`) as these properties will then be omitted during JSON serialization and I find that nicer than: `items:[]`